data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -194,88 +194,84 @@
     field_names.SCORE_L_COMMUNITIES,
     field_names.TOTAL_POP_FIELD,
     field_names.FPL_200_SERIES,
-    field_names.POVERTY_LESS_THAN_200_FPL_FIELD,
-    field_names.POVERTY_LESS_THAN_200_FPL_FIELD
-    + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD,
+    field_names.EXPECTED_AGRICULTURE_LOSS_RATE_LOW_INCOME_FIELD,
     field_names.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD
     + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.EXPECTED_AGRICULTURE_LOSS_RATE_LOW_INCOME_FIELD,
-    field_names.EXPECTED_BUILDING_LOSS_RATE_FIELD,
+    field_names.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD,
+    field_names.EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME_FIELD,
     field_names.EXPECTED_BUILDING_LOSS_RATE_FIELD
     + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME_FIELD,
```

Comment:
I think we're now missing `field_names.EXPECTED_BUILDING_LOSS_RATE_FIELD`?

Reply:
Oh, it looks like it's below on line 207. I think the building loss rate should be clustered with the other building loss rate fields here.

Reply:
Done